src/modules/item/selectors.spec.ts

```diff
@@ -111,4 +116,126 @@ describe('Item selectors', () => {
       expect(getRarities(state)).toEqual(state.item.rarities)
     })
   })
+
+  describe('when getting status by item id', () => {
+    it('should return the status for each published item', () => {
+      mockGetChainIdByNetwork.mockReturnValue(ChainId.MATIC_MAINNET)
```

Comment:
`.mockReturnValueOnce`

Reply:
This one i had to change it back to the `mockReturnValue` because it's called more than once (three times in this test, once of each item).